src/core/Akka/IO/Tcp.cs

```diff
@@ -830,7 +834,11 @@ public class WritingResumed : Event
             /// <summary>
             /// TBD
             /// </summary>
-            public static WritingResumed Instance = new WritingResumed();
+            public static readonly WritingResumed Instance = new WritingResumed();
+
+            private WritingResumed()
```

Comment:
These are breaking API changes, but no big deal. Need to be able to do stuff like this in Akka.NET v1.5.